config.toml

```diff
@@ -64,7 +64,7 @@ lastmod = ["lastmod", ":git", "date"]
   weight = 10
 
 [[menu.main]]
-  url = "/about"
+  url = "/syna"
```

Comment:
Shouldn't this be https://about.okkur.org/syna instead?

Reply:
I forgot to change it back. Shouldn't we remove the menus? As okkur website will override them. Or am I wrong?

Reply:
It won't be merged as far as my thinking goes as it is on a separate subdomain and with a separate netlify process.